climakitaegui/core/data_interface.py

```diff
@@ -627,7 +626,7 @@ def _display_select(self):
         pn.Row(col_1_location, col_2_location),
         title="Location Options for the Selected Data",
         collapsible=False,
-        width=600,
+        width=615,
```

Comment:
```suggestion
        width=635,
```

Reply:
I only added this suggestion (and the one below) to change the sizing on the stations list since I think you were working on changing it a bit in this PR. I noticed the lat/lon bars were a bit close to the station names, and that the station names were a bit close to the border, so I suggest the above and adding this margin for `col_2_location`:

```
    col_2_location = pn.Column(
        pn.Spacer(height=10),
        pn.widgets.StaticText(
            value="",
            name="Stations",
        ),
        pn.widgets.CheckBoxGroup.from_param(self.param.stations, name=""),
        width=300,
        margin=(0, 0, 0, 15),
    )
```

Feel free to disregard these 2 suggestions if you'd like as well!

Reply:
Image I was going off of:

<img src="https://github.com/user-attachments/assets/5b8f1137-ac01-459c-9cc0-17a616872a78" alt="Alt Text" width="380" height="700">